atlasdb-config/src/test/java/com/palantir/atlasdb/factory/LeadersTest.java

```diff
@@ -41,7 +43,8 @@
 
 public class LeadersTest {
 
-    public static final Set<String> REMOTE_SERVICE_ADDRESSES = ImmutableSet.of("foo:1234", "bar:5678");
+    private static final Set<String> REMOTE_SERVICE_ADDRESSES = ImmutableSet.of("foo:1234", "bar:5678");
+    private static final Supplier<RemotingClientConfig> REMOTING_CLIENT_CONFIG = () -> RemotingClientConfig.ALWAYS_USE_LEGACY;
```

Comment:
These tests are not the best, given that it passes with the other RCC...

Reply:
I don't even know what this is supposed to test, it's all broken mocks

Reply:
This test just validates certain configurations (e.g. that a list of Paxos learners will include the local learner). The test doesn't actually test any CJR or Feign functionality, just that a remoting configuration is necessary.

I might as well make this CJR then.